apps/portal/src/pages/[id]/manager-approvals/overtimes.tsx

```diff
@@ -150,6 +150,12 @@ export default function OvertimeApprovals({ employeeDetails }: InferGetServerSid
     }
   }, [patchResponseOvertime]);
 
+  useEffect(() => {
+    if (!pendingOvertimeModalIsOpen) {
+      setSelectedOvertimeId(''); //reset selected OT ID for OT modal content
+    }
+  }, [pendingOvertimeModalIsOpen]);
+
   const photoUrl_temp = '/profile.jpg';
```

Comment:
suggestion to add the profile.jpg to import instead of a constant variable

Reply:
Resolved in the next commit
Added 
images: {
    domains: ['172.20.110.45'],
  },
in next.config.js to allow fetching of image in this domain/ip